metastore/scripts/upgrade/hive/hive-schema-4.0.0-alpha-2.hive.sql

```diff
@@ -1138,6 +1144,9 @@ CREATE EXTERNAL TABLE IF NOT EXISTS `COMPLETED_COMPACTIONS` (
   `CC_HIGHEST_WRITE_ID` bigint,
   `CC_HADOOP_JOB_ID` string,
   `CC_ERROR_MESSAGE` string,
+  `CC_NEXT_TXN_ID` bigint,
```

Comment:
I think, CC_NEXT_TXN_ID` & `CC_TXN_ID are only relevant for in-flight processing and not useful during the troubleshooting (txnIds would be already gone)

Reply:
Well, just thinking about troubleshooting situations... txnIds will be present in the logs and might be still left open in the TXNS table. NEXT_TXN_ID will tell you when the cleaner should have picked up the compaction for cleaning.
Do you think there's any downside to propagating them?